exchanges/huobi/huobi_wrapper.go

```diff
@@ -288,16 +289,30 @@ func (h *HUOBI) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.
 			return nil, err
 		}
 		pairs = make([]currency.Pair, 0, len(symbols.Data))
-		for c := range symbols.Data {
-			if symbols.Data[c].ContractStatus != 1 {
+		expiryCodeDates := map[string]currency.Code{}
+		for _, c := range symbols.Data {
+			if c.ContractStatus != 1 {
 				continue
 			}
-			pair, err := currency.NewPairFromString(symbols.Data[c].ContractCode)
+			pair, err := currency.NewPairFromString(c.ContractCode)
```

Comment:
```suggestion
			pair, err = currency.NewPairFromString(c.ContractCode)
```

Reply:
Imma do the opposite, hope that's okay.
As you know, I always try to keep var scope as small as possible.

Fixed 85442e6a